server/.env.example

```diff
@@ -0,0 +1,7 @@
+MONGODB=mongodb+srv://cheesetouched:68A2sS8P5XSv6Ur@hackathon.ea6nb.mongodb.net/onboardio?retryWrites=true&w=majority
```

Comment:
Is this file supposed to be committed?

Reply:
I think this got committed by mistake @anje123 Can you remove it from the PR?

Reply:
ok bro